### ISSUE TITLE
Posters in mailbox

### ISSUE THREAD

**Issue by yiwenteng** [IITIDIDX597/F2023_Team1#33]
Goal statement:
For the UChicago Medicine HR team who wants to increase registration rates for the MyWellbeing employee wellness program by 24%, the MWB communication toolkit that has all the engagement components to increase awareness + staff engagement with MWB so that more employees can engage with the health resources they need to be as healthy as possible.

Hill statement:
UCM employees with limited opportunity to check email during their workday are able to learn about MWB in a manner that is convenient for them so that they can utilize ALL the free wellness resources they are entitled to.

Subhill statement:
Create a physical MWB marketing campaign specifically for employees with limited opportunity to check email.

User story: 
As an employee who infrequently checks email,
I want to receive physical posters in my mailbox periodically,
So that I am directly informed about the MWB program and its benefits, ensuring I don't miss out due to digital oversights.

Acceptance criteria: 
Given that I infrequently check email and may miss digital communications,
When I receive a physical MWB poster in my employee mailbox,
Then it should contain a clear and succinct summary of the MWB program and its benefits, using bullet points or infographics for quick reading, to ensure I get all the necessary information at a glance without relying on digital notifications.

**Comment by akallish** [comment]
@yiwenteng @emerydonovan I don't think it is realistic for a person to get a poster in a mailbox. This story seems highly implausible

**Comment by yiwenteng** [comment]
Criteria:
Given that I infrequently check email and may miss digital communications about the MWB program,
When I visit common areas where notices are typically posted, like break rooms, staff bulletin boards, or near time clocks,
Then there should be compact and informative MWB flyers displayed.

**Comment by akallish** [comment]
@yiwenteng This story is about noticing the poster.

I am not sure what the flyer is as these stories all seem focused on the poster.

What I do not see is any stories about a call to action like a QR code, etc. How does a person do something when they see the information on the poster?

Also, am not sure but this acceptance criteria and story should be in the original story issue so I can se the goal, hill statement and sub-hills for tracibility.

**Comment by jieanjie** [comment]
User Story:
As an employee who seldom checks emails, I want to receive clear MWB program posters in my office mailbox from time to time, ensuring I'm informed about the program and its benefits without missing details due to my limited email use.

Given that I seldom check emails and might overlook online messages,
When I find an MWB program poster in my office mailbox,
Then it should present a brief yet informative summary of the MWB program, highlighting key benefits.

**Comment by akallish** [comment]
@jieanjie I like your When statement and @yiwenteng Then statement.

**Comment by lidajiang19** [comment]
User story:
As an employee who infrequently checks email,
I want to receive physical posters in my mailbox periodically,
So that I am directly informed about the MWB program and its benefits, ensuring I don't miss out due to digital oversights.

A/C:
Given that I do not check emails frequently,
When I find an MWB program poster in my office mailbox,
Then there should be compact and informative MWB flyers displayed.

**Comment by akallish** [comment]
@lidajiang19 @jieanjie @yiwenteng @emerydonovan I am stil somewhat skeptical that a poster can be in a mailbox vs. a letter sized mailer that looks like the posters and is capturing much of the same information with a clear call to action which is to register. Am I missing something here?

**Comment by emerydonovan** [comment]
User story:
As an employee who infrequently checks email,
I want to receive an 8.5x11 flyer in my mailbox that has information about MWB and includes QR code 
So that I can easily register for the program without having to check my email.

A/C:
Given that I do not check emails frequently,
When I find an MWB program poster in my office mailbox,
Then I should be able to quickly register via the QR code on the physical poster.

**Comment by jieanjie** [comment]
Final：
Story
As an employee who infrequently checks email,
I want to receive physical posters in my mailbox periodically,
So that I am directly informed about the MWB program and its benefits, ensuring I don't miss out due to digital oversights.

A/C
Given that I infrequently check email and may miss digital communications,
When I receive a physical MWB poster in my employee mailbox,
Then it should contain a clear and succinct summary of the MWB program and its benefits, using bullet points or infographics for quick reading, to ensure I get all the necessary information at a glance without relying on digital notifications.
